api/openapi.yaml

```diff
@@ -353,6 +353,42 @@ paths:
           description: Endpoint not found
         '405':
           description: 'Invalid method, only GET,POST is allowed'
+
+  /bmc/creds:
+    parameters:
+      - in: query
+        name: targets
+        required: false
+        description: Comma separated list of XNames.  No query string results in fetching creds for all known BMCs.
+        schema:
+          $ref: '#/components/schemas/xname_list'
+      - in: query
```

Comment:
will PDUs show up? and I assume it wouldnt be their 'real' creds but their fake RF creds?